libs/platform/ebpf_object.h

```diff
@@ -29,6 +29,7 @@ extern "C"
         ebpf_object_type_t type;
         ebpf_free_object_t free_function;
         ebpf_list_entry_t entry;
+        ebpf_list_entry_t list_entry;
```

Comment:
`entry` and `list_entry` are both list entries.  Can we use more descriptive names to distinguish them?

Reply:
Changed to `global_list_entry` and `object_list_entry` and added comments about their usage.